content/articles/2018-02-forcats-0-3-0.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "forcats 0.3.0"
+slug: forcats-0-3-0
+description: >
+    A new version of forcats is now on CRAN!
+date: 2018-02-20
+photo:
+  url: https://www.pexels.com/photo/photo-of-a-cat-sleeping-on-gray-concrete-bench-757457/
+  author: Nicole Law
+categories: [package]
+---
+
+```{r setup, include = FALSE}
+library(forcats)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are pleased to announce that [forcats 0.3.0](http://forcats.tidyverse.org/) is now on CRAN. 
+
+The forcats package provides a suite of useful tools that solve common problems with factors in R. This version includes updates that resolve several of the "most popular" GitHub issues (as measured by reactions), some of which are highlighted below. For a complete set of changes, please see the [release notes](https://github.com/tidyverse/forcats/releases/tag/v0.3.0).
+
+You can install the latest version with:
+
+```{r, eval = FALSE}
+install.packages("forcats")
+```
+
+### Improvements to NA handling
+
+Closing out one of the most up-voted [issues](https://github.com/tidyverse/forcats/issues/89) in the tidyverse, `as_factor()` and `fct_inorder()` can accept NA levels. 
```

Comment:
Maybe a small example here? 

(And would remind people what `fct_inorder()` does, because it's something a lot of people seem to want)